Video Speed Controller/main.js

```diff
@@ -0,0 +1,17 @@
+const speed = document.querySelector('.speed');
+const bar = speed.querySelector('.speed-bar');
+const video = document.querySelector('.flex');
+
+speed.onclick = function handleMove(e) {
+    const y = e.pageY - this.offsetTop;
+    const percent = y / this.offsetHeight;
+    const min = 0.4;
+    const max = 4;
+    const height = Math.round(percent * 100) + '%';
+    const playbackRate = percent * (max - min) + min;
+    bar.style.height = height;
+    bar.textContent = playbackRate.toFixed(2) + '×';
+    video.playbackRate = playbackRate;
+}
+
+speed.addEventListener("mousemove", handleMove);
```

Comment:
pay attention to the error in console **Uncaught ReferenceError: handleMove is not defined**

Reply:
done